src/Update.php

```diff
@@ -259,7 +259,16 @@ function (string $flag) {
         $migrations = $this->getMigrationsToDo($current_version, $force_latest);
         foreach ($migrations as $key => $migration_specs) {
             include_once($migration_specs['file']);
-            $migration_specs['function']();
+
+            try {
+                $migration_specs['function']();
+            } catch (\Throwable $e) {
+                $this->migration->displayError(sprintf(
+                    __('An error occurred during the update. The error was: %s'),
+                    $e->getMessage()
+                ));
+                die(1);
```

Comment:
Aren't we supposed to drop all `die` and similar calls? (I do not have in mind the process)

Reply:
Yes, but this one requires many changes, and this PR is already big enough.

Reply:
ok